packages/entity-browser/src/components/EntityBrowser/EntityBrowser.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import * as ToccoUI from 'tocco-ui'
 import './styles.scss'
+import {InputFactory} from './InputFactory'
 
 export class EntityBrowser extends React.Component {
```

Comment:
We should use stateless functional components instead of classes wherever possible.

See also https://medium.com/@housecor/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc#.c4yqbjs15